### PR TITLE
Teshari cultural food

### PR DESCRIPTION
**Pull request by MidoriWroth**
## About The Pull Request

This PR adds a series of teshari cultural food and plants. The list isn't too massive currently, as I simply wanted to get this damn thing done, but I'm pretty pleased with it so far.

Some things were placed in the TG files instead of the modular_skyrat files simply because I couldn't find a suitable place for them in the skyrat files. If you wanna make better suggestions for where to put things, I wouldn't mind.

Also ignore the blue glow from the nakati bark, it wasn't supposed to be like that and I forgot to remove it and didn't want to put that whole picture together again lmao

This PR will add:

- 12 new teshari food recipes, in their own defined list in the crafting menu
- 4 new teshari-type plants and their subsequent liquids/powders used for teshari cooking

image

## How This Contributes To The Skyrat Roleplay Experience

This is an idea I have had for a while, and finally got around to it. Giving teshari their own cultural food allows for a greater diversity in not only chef cooking, but gives people a chance to roleplay around the food.

## Changelog

:cl:
add: Adds teshari cultural food!
add: Adds 4 new plants used for the teshari food
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Holy shit the madlad actually did it

File: code/__DEFINES/construction.dm [hunk 1]
```diff
@@ -107,6 +107,9 @@
 #define CAT_SANDWICH "Sandwiches"
 #define CAT_SOUP "Soups"
 #define CAT_SPAGHETTI "Spaghettis"
+		// SKYRAT ADDITION BEGIN - Teshari food define
+#define CAT_TESHARI "Teshari Food"
+		// SKYRAT ADDITION END
```
Comment (tf-4) [inline code/__DEFINES/construction.dm]: `code/__DEFINES/~skyrat_defines/construction.dm`

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 2]
```diff
@@ -1076,3 +1076,40 @@
...
+
+/datum/reagent/consumable/nakati_spice
+	name = "Nakati Spice"
+	description = "A coarse brown powder ground from nakati bark which serves as a staple in teshari cooking. The aroma is delightful, but extremely potent."
+	nutriment_factor = 2 * REAGENTS_METABOLISM
+	color = "#735c45"
+	taste_mult = 3
+	taste_description = "potent seasoning"
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
+		// SKYRAT ADDITION END
```
Comment (tf-4) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]: `modular_skyrat/modules/customization/modules/reagents/chemistry/reagents/food_reagents.dm`
(this is not really a great place but it's the best currently available)

Comment (MidoriWroth) [reply code/modules/reagents/chemistry/reagents/food_reagents.dm]: Gotcha, I'll do that

**Comment by ghost** [comment]
Hello hello it is sprite review time!! _None is official nor matters but if you do it it may look nicer in the end but I could also be wrong so dont have 100% faith in me its 230am_ :3

Everything looks good, only a few notes:
- The Piru items have a lot of random mottling, the dough/flattened/flatbread being the worst offenders - I'd recommend using recolored versions of TG's bread icons as a base for most of them and reshaping them a bit (See icons/obj/food/food_ingredients.dmi)
- The Piru frond and Kiri fruit could be more consistent with their final growth stage (mostly coloring-wise), as they're harvested as a whole item (Which is actually how they function per the Avali lore, so I'm glad that that's how they work in this PR too!)
- The jerky desperately needs tweaking, it looks the opposite of appetizing - mostly, though, that's the fault of the base sprite under the seasoning. I'd recommend using the original cutlet sprite with the Goliath Steak's palette, it makes it look more meaty, less dry, and spicier!:
image
(The black background makes it look worse it does look how I describe it I swear)

I have to say though, the Baked Kiri fruit looks legitimately delicious mmmghbngh

**Comment by EerieDream** [comment]
Yes.

**Comment by HoodMilk** [comment]
Might be a good idea to rename them so they aren't a direct copy of avali stuff.

**Comment by MidoriWroth** [comment]
> Hello hello it is sprite review time!! _None is official nor matters but if you do it it may look nicer in the end but I could also be wrong so dont have 100% faith in me its 230am_ :3
> 
> Everything looks good, only a few notes:
> 
>     * The Piru items have a lot of random mottling, the dough/flattened/flatbread being the worst offenders - I'd recommend using recolored versions of TG's bread icons as a base for most of them and reshaping them a bit (See icons/obj/food/food_ingredients.dmi)
> 
>     * The Piru frond and Kiri fruit could be more consistent with their final growth stage (mostly coloring-wise), as they're harvested as a whole item (Which is actually how they function per the Avali lore, so I'm glad that that's how they work in this PR too!)
> 
>     * The jerky desperately needs tweaking, it looks the opposite of appetizing - mostly, though, that's the fault of the base sprite under the seasoning. I'd recommend using the original cutlet sprite with the Goliath Steak's palette, it makes it look more meaty, less dry, and spicier!:
>       image
>       (The black background makes it look worse it does look how I describe it I swear)
> 
> 
> I have to say though, the Baked Kiri fruit looks legitimately delicious mmmghbngh

For the breads, I made them that way sort of because kiri jelly is glittery and sparkly by nature, so the raw dough kinda reflects that while the baked dough is still somewhat glittery but in a more burnt sense.

After looking back at the jerky yeah i agree lmao, so I redid it with more reds, i think it looks better 
image

For the piru and kiri, I dunno what you mean, they look pretty similar to their growth stages, but I suppose that can be tweaked

**Comment by MidoriWroth** [comment]
> Might be a good idea to rename them so they aren't a direct copy of avali stuff.

Teshari are actually part of the official avali lore, so I don't think it will matter too much =)

**Comment by ghost** [comment]
> For the piru and kiri, I dunno what you mean, they look pretty similar to their growth stages, but I suppose that can be tweaked

It's very minor, but the shading between the final stage and the product itself is different; the goal would be the sprites being about identical, so it's as if you literally plucked them right out of the tray

**Comment by weesnaw-ops** [comment]
change the name from 'crisps' to 'chips'

**Comment by tf-4** [comment]
but they're not chips

File: code/__DEFINES/~skyrat_defines/construction.dm [hunk 3]
```diff
@@ -0,0 +1,3 @@
+		// SKYRAT ADDITION BEGIN - Teshari food define
```
Comment (jjpark-kb) [inline code/__DEFINES/~skyrat_defines/construction.dm]: its already in a skyrat_define folder, you don't need these.

File: code/datums/components/crafting/crafting.dm [hunk 4]
```diff
@@ -42,6 +42,9 @@
 					CAT_SANDWICH,
 					CAT_SOUP,
 					CAT_SPAGHETTI,
+		// SKYRAT ADDITION BEGIN - Teshari food category
+					CAT_TESHARI,
+		// SKYRAT ADDITION END
```
Comment (jjpark-kb) [inline code/datums/components/crafting/crafting.dm:47]: I'm not saying no, but we do need to find a better way of modularly doing this!

File: code/__DEFINES/~skyrat_defines/construction.dm (hunk 3 above)
Comment (MidoriWroth) [reply code/__DEFINES/~skyrat_defines/construction.dm]: what do you mean? just doing what @tf-4 suggested earlier

Comment (tf-4) [reply code/__DEFINES/~skyrat_defines/construction.dm]: Mhm, but now that it's in a skyrat folder you don't need the comments. Cus you only need them on core files.

Comment (MidoriWroth) [reply code/__DEFINES/~skyrat_defines/construction.dm]: ahhh right right right

**Pull request merged**